superflore/generators/ebuild/ebuild.py

```diff
@@ -157,13 +157,11 @@ def get_ebuild_text(self, distributor, license_text, die_msg=None):
         ret += " license\n\n"
 
         # EAPI=<eapi>
-        ret += "EAPI=" + self.eapi + "\n\n"
-        """
-        @todo: don't hard code this
-        """
+        ret += "EAPI=" + self.eapi + "\n"
+        ret += "PYTHON_COMPAT=( python{2_7,3_5} )\n\n"
```

Comment:
> it will still install all packages (as exhibited by my ability to install ros-lunar with 3.5)

If you count silent failures then you "installed" it, the thing that bother me more is that we will never see if a package is building fine or not because the installation process will continue to run in case of errors. This needs to be fixed upstream.